app/lib/three_scale/redis_config.rb

```diff
@@ -71,6 +66,12 @@ def parse_sentinels(sentinels)
       end
     end
 
+    def set_pool_size(config)
+      return if config[:size].present?
+
+      config[:size] = config.delete(:pool_size) if config.key?(:pool_size)
+    end
```

Comment:
We don't care about the `:pool_size` param anymore, but we don't want to break customers configurations, so `:pool_size` is accepted for backwards compatibility but transformed to `:size` internally.